config/initializers/assets.rb

```diff
@@ -1 +1 @@
-Rails.application.config.assets.precompile += ['guide/application.js', 'guide/scenario.js']
+Rails.application.config.assets.precompile += %w[guide/application.js guide/scenario.js guide/application.css]
```

Comment:
The only change required is explicitly reference the CSS included with the gem.

Reply:
Is this how gems are meant to configure assets they use with sprockets, or does sprockets support `app/assets/config/manifest.js` file for gems as well?

Reply:
Good question.

I see in the Rails Guide it configures things a little differently:

```ruby
initializer "blorgh.assets.precompile" do |app|
  app.config.assets.precompile += %w( admin.js admin.css )
end
```
https://guides.rubyonrails.org/engines.html#separate-assets-and-precompiling

I'll try moving to this form.